.github/CODEOWNERS

```diff
@@ -171,6 +171,9 @@
 /src/permission/* @nodejs/security-wg
 /test/parallel/test-permission-* @nodejs/security-wg
 
+# Security Release
+/doc/contributing/security-release-process.md @nodejs/security-release
```

Comment:
I'm not sure this is the right group for codeowners. The people who follow this process are the [security stewards](https://github.com/nodejs/node/blob/main/doc/contributing/security-release-process.md#security-release-stewards), most of whom are not members of security-release (which is more or less releasers).

Reply:
(There is a team for the security stewards, but only in the private org. -- I'm not sure if that's an oversight because the other `security-*` teams are replicated between the two orgs.)

Reply:
Should I change it to `@nodejs/security`? We don't have a security stewards group but we could create if needed.

Reply:
It might make sense to use the existing nodejs/security team, however, we do want the security stewards to be in the code owners and I don't think that would do it since we don't have them in a team, and its not certain (not sure on this) they are in the other security teams?

Reply:
Maybe it's worth-it to have a @nodejs/security-stewards, WDYT?

Reply:
I'm + to adding @nodejs/security-stewards